src/chains/data.ts

```diff
@@ -235,6 +235,37 @@ export const chiliz: Sablier.Chain = define(
   }),
 );
 
+export const hyperevm: Sablier.Chain = define(
```

Comment:
It would be helpful to mention the chain ID conflict in an explanatory comment:

https://github.com/wevm/viem/pull/3390

Reply:
Defining custom chains like this means additional maintenance costs for us, so we should always be on the lookout to remove the custom definitions, when possible. For instance, Chiliz and Morph have the Multicall3 address listed now, so we can remove them: https://github.com/sablier-labs/sdk/issues/16

I found it surprising that a chain as big as HyperEVM wasn't in Viem, so I went and looked for it, and found out about the collision.

Reply:
I agree. I can see you have already pushed a commit. Thank you.